src/test/rgw/lua/test_lua.py

```diff
@@ -0,0 +1,475 @@
+import logging
+import json
+import tempfile
+import random
+import socket
+import time
+import threading
+import subprocess
+import os
+import stat
+import string
+import pytest
+import boto3
+
+from . import(
+    get_config_host,
+    get_config_port,
+    get_access_key,
+    get_secret_key
+    )
```

Comment:
in https://github.com/ceph/s3-tests/pull/482/commits/672a1233485926b1e9fdcb4170cb7b4e3abcb30b, it was necessary  to import the fixture to make it run. try adding `configfile` here?